### ISSUE TITLE
Opening "Discovered hosts" in the GUI causes an error

### ISSUE THREAD

**Issue by laugmanuel** [dm-drogeriemarkt/foreman_git_templates#60]
When [foreman_discovery](https://github.com/theforeman/foreman_discovery) is used, opening "Discovered hosts" in the GUI causes the following error:
```
ActionView::Template::Error
undefined method `repository_path' for #<Host::Discovered:0x00007fd9617030b8>
/opt/theforeman/tfm-ror52/root/usr/share/gems/gems/activemodel-5.2.1/lib/active_model/attribute_methods.rb:430:in `method_missing'
/opt/theforeman/tfm/root/usr/share/gems/gems/audited-4.9.0/lib/audited/auditor.rb:99:in `method_missing'
/opt/theforeman/tfm/root/usr/share/gems/gems/foreman_git_templates-1.0.5/app/helpers/foreman_git_templates/application_helper.rb:11:in `display_link_if_authorized'
/opt/theforeman/tfm/root/usr/share/gems/gems/foreman_discovery-16.0.1/app/helpers/discovered_hosts_helper.rb:21:in `block in discovered_hosts_title_actions'
[...]
```

**Comment by timogoebel** [comment]
@laugmanuel: Can we close this?

**Comment by laugmanuel** [comment]
Closed by #61
